src/Http/Form/SignupForm.php

```diff
@@ -201,15 +202,29 @@ public function validateLastName(): bool
         return $validationResponse;
     }
 
+    public function validateJoindInUsername(): bool
+    {
+        if (!isset($this->cleanData['joindin_username'])
+            || $this->cleanData['joindin_username'] == ''
+            || \preg_match('/^[a-zA-Z0-9\-_\.]{1,100}$/', $this->cleanData['joindin_username'])
```

Comment:
Should we make sure we are dealing with a `string` here before applying `preg_match()`?